packages/auth/src/lib/createAuthToken.ts

```diff
@@ -1,26 +1,20 @@
 import { randomBytes } from 'crypto';
```

Comment:
should this be `node:crypto` instead similar to https://github.com/keystonejs/keystone/pull/8725/files#diff-26c287c98ecdad9f09205169e711de7a0e467c876d50deb5f89fa9e58ac15005R1

Reply:
Yes, but that shouldn't be a problem either

Reply:
just sanity check, no complain